adplug/rix.cpp

```diff
@@ -213,27 +225,27 @@ float CrixPlayer::getrefresh()
 }
 
 /*------------------Implemention----------------------------*/
-inline void CrixPlayer::set_new_int()
```

Comment:
Oh, in fact its removed for ease of debugging, forgot restoring them after debugging. A macro is a good suggestion in debug purpose.